pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiQuestionValidation.java

```diff
@@ -83,6 +83,9 @@ Stream<RuleDto> findDtosByQuestionIdsAndTimestamp(
             @BindList(value = "questionIds", onEmpty = EmptyHandling.NULL) Iterable<Long> questionIds,
             @Bind("timestamp") long timestamp);
 
+    @SqlUpdate("DELETE FROM question__validation WHERE question_id = :questionId")
+    boolean deleteForQuestion(@Bind("questionId")long questionId);
```

Comment:
whitespace after the annotation

```suggestion
    boolean deleteForQuestion(@Bind("questionId") long questionId);
```